suite-native/module-home/src/components/Assets.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { AssetItem, Card, VStack } from '@suite-native/atoms';
+
+import { DashboardSection } from './DashboardSection';
+
+export const Assets = () => (
+    <DashboardSection title="Assets">
+        <Card>
+            <VStack spacing={19}>
+                <AssetItem
```

Comment:
Progress bars there should share the main color of the coin, perhaps should be in some config?